Clean up folder structure and purge

Magnus93 opened the pull request:
- accordion
- display-date-time
- popup
- reorder
- skeleton
- svg
- tuple
- urlencoded
- and more...

AntonCroner reviewed (changes requested):
Label is garbage (made by me before i knew anything) but still needed.

Magnus93 committed:
Purge old and unused components

Purged
- accordion
- display-date-time
- popup
- reorder
- skeleton
- svg
- tuple
- urlencoded

Magnus93 committed:
Remove address related iinputs

Magnus93 committed:
FIx tabs and app demo

Magnus93 committed:
Fix dialog and display demo folder structure

Magnus93 committed:
deprecate display amount

Magnus93 committed:
Fix folder structure for icon and trigger.

Magnus93 committed:
Fix name on toggle file.

Magnus93 committed:
Fix button folder structure.

Magnus93 committed:
Deprecate picker
